src/components/ButtonPanel.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import Button from './Button'
+
+function ButtonPanel() {
+  return (
+    <div className="button-panel">
+      <div className="group1">
+        <button names="AC" />
+        <button names="+/-" />
+        <button names="%" />
+        <button names="AC" />
+        <button names="&#247;" />
```

Comment:
- [ ] The first group should be: `AC, +/-, %, ÷`. We have `AC` appearing two times